v1.18/eks-d/README.md

```diff
@@ -0,0 +1,190 @@
+    --node-size m5.xlarge \
+    --kubernetes-version https://distro.eks.amazonaws.com/kubernetes-1-18/releases/1/artifacts/kubernetes/v1.18.9 \
+    --master-size m5.large \
+    --dry-run \
+    -o yaml > $CLUSTER_NAME.yaml
+echo "Add the following content to your Cluster spec in $CLUSTER_NAME.yaml"
+echo
+cat << EOF >> /dev/stdout
+  kubeAPIServer:
+    image: public.ecr.aws/eks-distro/kubernetes/kube-apiserver:v1.18.9-eks-1-18-1
+  kubeControllerManager:
+    image: public.ecr.aws/eks-distro/kubernetes/kube-controller-manager:v1.18.9-eks-1-18-1
+  kubeScheduler:
+    image: public.ecr.aws/eks-distro/kubernetes/kube-scheduler:v1.18.9-eks-1-18-1
+  kubeProxy:
+    image: public.ecr.aws/eks-distro/kubernetes/kube-proxy:v1.18.9-eks-1-18-1
+  # Metrics Server will be supported with kops 1.19
+  metricsServer:
+    enabled: true
+    image: public.ecr.aws/eks-distro/kubernetes-sigs/metrics-server:v0.4.0-eks-1-18-1
+  authentication:
+    aws:
+      image: public.ecr.aws/eks-distro/kubernetes-sigs/aws-iam-authenticator:v0.5.2-eks-1-18-1
+  kubeDNS:
+    provider: CoreDNS
+    coreDNSImage: public.ecr.aws/eks-distro/coredns/coredns:v1.7.0-eks-1-18-1
+    externalCoreFile: |
+      .:53 {
+          errors
+          health {
+            lameduck 5s
+          }
+          kubernetes cluster.local. in-addr.arpa ip6.arpa {
+            pods insecure
+            #upstream
+            fallthrough in-addr.arpa ip6.arpa
+          }
+          prometheus :9153
+          forward . /etc/resolv.conf
+          loop
+          cache 30
+          loadbalance
+          reload
+      }
+  masterKubelet:
+    podInfraContainerImage: public.ecr.aws/eks-distro/kubernetes/pause:v1.18.9-eks-1-18-1
+  # kubelet might already be defined, append the following config
+  kubelet:
+    podInfraContainerImage: public.ecr.aws/eks-distro/kubernetes/pause:v1.18.9-eks-1-18-1
+EOF
+```
+
+After you have finished configuring your cluster, run the update:
+
+```
+# For more details see: https://github.com/kubernetes/test-infra/pull/14731
+export KOPS_FEATURE_FLAGS=SpecOverrideFlag
+./kops set cluster ${CLUSTER_NAME} cluster.spec.nodePortAccess=0.0.0.0/0
+
+# At this point, you can configure your cluster as you would like.
+# By design, kops gives you many options on different options to configure and bootstrap
+# your clusters. The conformance cluster is a simple cluster but conformance should pass with
+# more complex configurations as well.
+#
+# For more information see docs: https://kops.sigs.k8s.io/cluster_spec/
+
+./kops update cluster ${CLUSTER_NAME} --yes
+```
+
+Wait for the cluster to come up which may take 5-10 minutes.
+
+Running `./kops validate cluster` allows you to see when the cluster is up and ready to run conformance.
+
+## Run Sonobuoy e2e
+```
+./sonobuoy run --mode=certified-conformance --kube-conformance-image k8s.gcr.io/conformance:v1.18.9
+results=$(./sonobuoy retrieve)
+mkdir ./results
+tar xzf $results -C ./results
+./sonobuoy e2e ${results}
```

Comment:
I think you can remove this

Reply:
I've only looked at the kops and eks entries, but the both cover results like that. I assume it is required.

Reply:
youre right!